streams/src/test/java/org/apache/kafka/streams/processor/internals/StateManagerUtilTest.java

```diff
@@ -190,31 +167,7 @@ public void testCloseStateManagerClean() throws IOException {
     }
 
     @Test
-    public void testCloseStateManagerThrowsExceptionWhenClean() throws IOException {
```

Comment:
Removed `testCloseStateManagerOnlyThrowsFirstExceptionWhenClean` since there's now only one possible source of exceptions in the clean path, and the below test `testCloseStateManagerOnlyThrowsFirstExceptionWhenClean` covers this case